presto-native-execution/presto_cpp/main/TaskManager.h

```diff
@@ -42,6 +42,8 @@ class TaskManager {
 
   void setBaseSpillDirectory(const std::string& baseSpillDirectory);
 
+  bool emptyBaseSpillDirectory() const;
```

Comment:
What's this used for? Can we remove? thanks!

Reply:
If a worker does not get spill path in start up, we want to schedule periodic task to update its spill path. Also we want to skip updating once it has a spill path. This method will tell the periodic task if a remote call is needed.